### DISCUSSION TITLE
Disable Clickable day

### DISCUSSION THREAD

**Discussion by hrodrigues-belodigital** [saade/filament-fullcalendar#179] [Q&A]
Hello, just started using this package.
I was wondering If I could somehow disable some days from being clickable and the events on them from being editable.
Ex: holidays, I don't want anyone to be able to create an event on those days and the event's in those days to not be able to be edited.

**Comment by SariMorgenshtern** [comment]
@hrodrigues-belodigital 
Were you able to disable clicking on selected days in the calendar view?
